### PR TITLE
better explanations for py3 switch

### PR DESCRIPTION
**Pull request by fchapoton**
once merged, please make a new release

### CODE REVIEW

File: sagenb_export/nbextension/list_handler.html [hunk 1]
```diff
@@ -18,10 +18,11 @@ <h2>Use the Jupyter notebook.</h2>
...
                 on the command line.
             </p>
-            <h2>The old notebook is now deprecated.</h2>            
+            <h2>The old notebook is now deprecated.</h2>
             <p>
-                The old Sage notebook only works with Python2 and is now deprecated.
-                It is still available as an optional package, that you may need to install using <tt>sage -i sagenb</tt>.
+                Starting from version 9.0, Sage is using Python3 by default.
+                The old Sage notebook only works for Sage using Python2 and is now deprecated.
+                The old Sage notebook is still available as an optional package, working only under Python2 and that you need to install using <tt>sage -i sagenb</tt>.
```
Comment (dimpase) [inline sagenb_export/nbextension/list_handler.html]: I suggest this small edit:

`...Python2 and that you...` -> `...Python2. You...`

**Comment by kcrisman** [comment]
Is there a place/link we can send people to in order to learn about how to use Sage on Py2?  In case anyone hasn't yet exported their worksheets.

**Comment by vbraun** [comment]
If they haven't exported their worksheets then now is a good time ;-) I don't have a particular link though.

**Comment by fchapoton** [comment]
Please edit and enhance the wiki page
https://wiki.sagemath.org/Python3-Switch

**Comment by fchapoton** [comment]
May we please now have a new release, so that we can update the package inside sage ?

**Comment by vbraun** [comment]
New release is at https://trac.sagemath.org/ticket/28693

**Pull request merged**